tests/test_jx/test_deep_ops.py

```diff
@@ -777,15 +777,15 @@ def test_aggs_on_parent(self):
                 "meta": {"format": "cube"},
                 "select": {"name": "count"},
                 "edges": [
-                    {"name": "o", "domain": {"type": "set", "partitions": [
+                    {"name": "o", "allowNulls": False, "domain": {"type": "set", "partitions": [
```

Comment:
The query, by default, demands the null coordinates to show, even if they are zero,